test_4_filter_numbers.py

```diff
@@ -1,6 +1,9 @@
 def filter_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
+  for i in arr[:]:
```

Comment:
Can you tell me more about your use of `arr[:]`? What was your thought process in including it?